packages/darklang/stdlib/list.dark

```diff
@@ -153,4 +153,300 @@ module Darklang =
           | head :: tail ->
             List.append_v0
               (List.singleton_v0 head)
-              (PACKAGE.Darklang.Stdlib.List.take tail (count - 1))
\ No newline at end of file
+              (PACKAGE.Darklang.Stdlib.List.take tail (count - 1))
+
+
+      // can we change the orderof the arguments?
+      ///Returns a list containing <param val> repeated <param times> times
+      let repeat
+        (times: Int)
+        (value: 'a)
+        : PACKAGE.Darklang.Stdlib.Result.Result<List<'a>, String> =
+        if times < 0 then
+          PACKAGE.Darklang.Stdlib.Result.Result.Error
+            $"Expected `times` to be positive, but it was `{times |> Int.toString}`"
+        else if times > 2147483647 then
+          PACKAGE.Darklang.Stdlib.Result.Result.Error
+            $"Expected `times` to be less than 2147483647, but it was `{times |> Int.toString}`"
+        else if times == 0 then
+          PACKAGE.Darklang.Stdlib.Result.Result.Ok []
+        else
+          match PACKAGE.Darklang.Stdlib.List.repeat (times - 1) value with
+          | Ok list ->
+            PACKAGE.Darklang.Stdlib.Result.Result.Ok(
+              PACKAGE.Darklang.Stdlib.List.pushBack list value
```

Comment:
pushBack is bad performance and should be avoided. Since all the values are the same, it will work the same if you push to the front.